traffic_ops/testing/api/v1/servers_test.go

```diff
@@ -71,19 +70,18 @@ func UpdateTestServers(t *testing.T) {
 	var alert tc.Alerts
 	alert, _, err = TOSession.UpdateServerByID(remoteServer.ID, remoteServer)
 	if err != nil {
-		t.Errorf("cannot UPDATE Server by hostname: %v - %v", err, alert)
+		t.Fatalf("cannot UPDATE Server by ID: %v - %v", err, alert)
 	}
-
 	// Retrieve the server to check rack and interfaceName values were updated
 	resp, _, err = TOSession.GetServerByID(remoteServer.ID)
 	if err != nil {
-		t.Errorf("cannot GET Server by ID: %v - %v", remoteServer.HostName, err)
+		t.Fatalf("cannot GET Server by ID: %v - %v", remoteServer.ID, err)
 	}
 
 	respServer := resp[0]
 	if respServer.InterfaceName != updatedServerInterface || respServer.Rack != updatedServerRack {
-		t.Errorf("results do not match actual: %s, expected: %s", respServer.InterfaceName, updatedServerInterface)
-		t.Errorf("results do not match actual: %s, expected: %s", respServer.Rack, updatedServerRack)
+		t.Fatalf("results do not match actual: %s, expected: %s", respServer.InterfaceName, updatedServerInterface)
+		t.Fatalf("results do not match actual: %s, expected: %s", respServer.Rack, updatedServerRack)
```

Comment:
Why are these fatal now? The rest of the test doesn't panic if the comparison is untrue. In fact, by making the first of these fatal, you've ensured that no one will ever see the second one, meaning that you can get the cryptic failure message:
```
results do not match actual: test, expected: test
```